server/main.lua

```diff
@@ -1011,8 +1011,8 @@ QBCore.Commands.Add("paytow", "Pay Tow Driver (Police, EMS Only)", {{name="id",
         local OtherPlayer = QBCore.Functions.GetPlayer(playerId)
         if OtherPlayer ~= nil then
             if OtherPlayer.PlayerData.job.name == "tow" then
-                OtherPlayer.Functions.AddMoney("bank", 500, "police-tow-paid")
-                TriggerClientEvent('chatMessage', OtherPlayer.PlayerData.source, "SYSTEM", "warning", "You received $ 500 for your service!")
+                OtherPlayer.Functions.AddMoney("bank", Config.BailJob, "police-tow-paid")
+                TriggerClientEvent('chatMessage', OtherPlayer.PlayerData.source, "SYSTEM", "warning", "You received $ 250 for your service!")
```

Comment:
This would need to be dynamic as the amount could be different. Checkout `string.format`